Code/Engine/Core/Utils/Blackboard.h

```diff
@@ -174,6 +174,14 @@ class EZ_CORE_DLL ezBlackboard : public ezRefCounted
   /// If deserialized entries overlap with existing ones, the deserialized entries will overwrite the existing ones (both values and flags).
   ezResult Deserialize(ezStreamReader& inout_stream);
 
+public:
+  // The following functions are for scripting reflection and shouldn't be called manually
```

Comment:
they don't need to be public, I think

Reply:
Unfortunately they need to be public for static reflected types, but I think I will move them to an extension class to clean up the interface here.

Reply:
Found a better way: EZ_ALLOW_PRIVATE_PROPERTIES